Apps/WebClient/src/ClientApp/src/models/errors.ts

```diff
@@ -28,15 +45,36 @@ export interface ResultError {
     actionCode?: ActionType;
```

Comment:
Suggest changing the documentation here to just "Action code." since the ResultError class is divorced from the RequestResult.